components/public-api/gitpod/v1/prebuild.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package gitpod.v1;
+
+import "google/protobuf/timestamp.proto";
+import "gitpod/v1/pagination.proto";
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
+
+service PrebuildService {
```

Comment:
@geropl could you have a look whether the definition makes sense to you?

I verified it against the dashboard code paths. I have not tested it.

Reply:
Done. Really cool to have this PrebuildService! :heart: :rocket: